content/articles/2018-05-ggplot2-2-3-0.Rmd

```diff
@@ -0,0 +1,126 @@
+are extracted from the problems report and put into a Google sheet where (with
+the great help of Thomas Lin Pedersen, Claus Wilke, and Kara Woo) we identified
+clusters of errors to tackle. For example, 97 packages returned a warning due
+to a namespace clash between `dplyr::vars()` and `ggplot2::vars()`. That’s
+something we clearly need to fix, so we open an [issue](https://github.com/
+tidyverse/ggplot2/issues/2585) to track progress.
+
+Where reverse-dependency errors result from a deliberate design change, we now
+have a collection of "symptomatic" error messages, and (hopefully) fixes to
+offer the maintainers of affected packages. These go in the "Breaking changes"
+section of the NEWS file, so that if you encounter a new error you can easily
+search for the error message and find the resolution. Because ggplot2 is an old
+and much used package, we try to keep breaking changes to a minimum. However,
+sometimes we deem it important to make a change that break existing code.
+Typically this is for one of two reasons:
+
+ * We discover that code that previously appeared to work was actually hiding a 
+ potential error. For example, in this version of ggplot2 we ensure that input 
+ data frames don’t have duplicate column names. Previously ggplot2 just took the 
+ first column with a name, which might have concealed a bug.  
+
+ * We decided that it’s worth breaking a small amount of existing code in the 
+ interests of improving future code. For example, this version of ggplot2 adds 
+ support for [tidy evaluation](https://github.com/tidyverse/ggplot2/blob/master/NEWS.md#tidy-evaluation) 
+ making ggplot2 more compatible with the rest of the tidyverse. This breaks a 
+ small number of packages that use ggplot2 but should affect very little data 
+ analysis code, so we decided that it was worth it.  
+
+### What to do if you run into a breaking change
+
+The maintainers of reverse dependencies that are likely to encounter a breaking 
+change will be contacted. However, it's always possible that we've missed 
+something. If you run into an error when running code that works with ggplot2 
+2.2.1 (the version on CRAN as of this writing), you should look for it in the
+["Breaking changes"](https://github.com/tidyverse/ggplot2/blob/master/NEWS.md#breaking-changes) 
+section of news. If you _can't_ find it in the news, please file an issue - 
+either it's an inadvertent change, or we need to describe it better in the news 
+(or make it easier to find when searching for keywords). As per usual, please 
+do a search of [existing issues](https://github.com/tidyverse/ggplot2/issues) 
+before filing a new one to avoid duplication.
+
+You can also ask for help in [community.rstudio.com](https://community.rstudio.com/).
+
+## New features
+
+### Tidy evaluation
+
+You can now use quasiquotation in `aes()`, `facet_wrap()`, and `facet_grid()`. 
+The formula interface for faceting is not going away, but this is meant to 
```

Comment:
Remove 2nd sentence and replace with: "To support quasiquotation in facetting we've added a new helper that works similar to `aes()`: `vars()`, short for variables and instead of `facet_grid(x + y ~  a + b)` you can now write `facet_grid(vars(x, y), vars(a, b))`. The formula interface won't go away; but the new `vars()` interface should be much easier to program with.